templates/base.html

```diff
@@ -19,18 +19,43 @@
         {% block extrastyle %}{% endblock %}
     </head>
     <body>
+    <div class="off-canvas-wrapper">
```

Comment:
I counted the `<div>` and I noticed you added 5 new open ones, but only closed 4 so one of these are off.